src/test/java/test/general/PartitionUtilsTest.java

```diff
@@ -39,7 +40,8 @@
 {
   private static final Logger logger = LoggerFactory.getLogger(PartitionUtilsTest.class);
 
-  public PartitionUtilsTest()
+  @BeforeClass
+  public static void setup()
   {
     PrimitiveTypePartitioner primitivePartitioner = new PrimitiveTypePartitioner();
   }
```

Comment:
This block of code can be removed, this instance of primitivePartitioner is not used in any of the tests.